modules/ank/arrow-ank/src/main/kotlin/arrow/ank/interpreter.kt

```diff
@@ -223,8 +223,7 @@ val interpreter: AnkOps = object : AnkOps {
           if (snippet.isPlayground)
             snippetContent.replace(snippet.fence, "{: data-executable='true'}\n\n```${snippet.lang}\n${snippet.code}\n```")
           else
-            snippetContent.replace(snippet.fence, "```${snippet.lang}\n${snippet.code}\n```")
-        },
+            snippetContent.replace(snippet.fence, "```${snippet.lang}\n${snippet.code}\n```") },
```

Comment:
This change makes no sense. Either fence the `else` clause with `{` or keep the other one on a new line

Reply:
A nit, not a blocker